docs/api/data.md

```diff
@@ -7,6 +7,244 @@ import WorkInProgress from '../components/WorkInProgress.mdx'
 
 # Data API
 
-A rich graphql based data API that can be used to query for any file metadata based on various query parameters.
+Data API provides a graphql based interface for storing and querying structured data. Decentralized application developers for Box can use this API to create, update and delete JSON documents using a standard graphql interface directly on their Box. The Data API is a part of [Borg Client](https://github.com/functionland/fula/tree/main/libraries/borg-client) and you can use it using the `graphql` interface.
```

Comment:
I recommend we use 'BOX' instead of 'Box'.  I think we are also getting rid of 'Borg' terminology  ( ? )   I recommend replacing it with 'Photos' client.

Reply:
The borg-client is going to be used in both `react-todo` and `Photos` examples. Once package renaming is done this also should get renamed.